sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -66,7 +66,7 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
                 'aggregated-runningExecutions')">
             <h4>
               <span class="collapse-table-arrow arrow-open"></span>
-              <a>Running Queries ({running.size})</a>
+              <a>Running Executions ({running.size})</a>
```

Comment:
The word `Executions` looks odd to me. 
How about just changing the tab name and leaving the word "Queries" as it is now? DataFrame operations can be considered as queries.

Reply:
> DataFrame operations can be considered as queries.

That's actually in dev perspective .. to end users, it's not really queries .. e.g., to Scala/Java/Python DataFrame users, SS users, ML users or pandas API on Spark users

Reply:
Will send an email to dev mailing list.

Reply:
I somewhat agree that executions sounds odd to me .   

Execution 0, Execution 1, Execution 2 makes me wonder, is that executing the same command just multiple times or is it different operations.  

I'm not sure I have a better idea then just leaving it Query though

Reply:
I guess you could also say more explicitly SQL/Dataframe Queries.
Another option programmatic/declarative queries, although that seems kind of long.

Reply:
Thanks for your suggestions. Let me think a bit more about it and your suggestions, and push some changes soon.